src/apps/nem/transfer/serialize.py

```diff
@@ -20,52 +25,54 @@ def serialize_transfer(
     payload: bytes = None,
     encrypted: bool = False,
 ) -> bytearray:
-    tx = write_common(
+    tx = serialize_tx_common(
         common,
-        bytearray(public_key),
+        public_key,
```

Comment:
serialize_tx_common expects [bytearray](https://github.com/trezor/trezor-core/pull/329/files#diff-86afd8d197d0e672da6c3180a7bf85e1R8) but here the `public_key` argument is defined as bytes. Is this native conversion okay?

Reply:
If we use bytearray only for reading, bytes are OK.

Reply:
:+1: